src/sample.config.json

```diff
@@ -5,6 +5,7 @@
   "port": 8080,
   "NODE_ENV": "development",
   "webhookURL": "",
+  "ngrokURL": "",
```

Comment:
Can you rename this to something like `serverURL`? Because after we host this in a VPS, it won't really be using ngrok. Also means we won't be needing `redirectDevURL` and `redirectProdURL` from below.

Reply:
Wouldnt it be better to change it when we are moving to VPS to avoid confusion.

Reply:
Hmmm, okay. We can add it as a separate variable that'll be used in production only I guess.